charts/pulsar/values.yaml

```diff
@@ -988,6 +988,24 @@ grafana:
     requests:
       memory: 250Mi
       cpu: 0.1
+  volumes:
+    # use a persistent volume or emptyDir
+    persistence: true
```

Comment:
What is the reasoning for making this the default behavior? Alternatively, we could mount dashboards as config maps and avoid the extra storage costs associated with a PVC.

Reply:
Just thinking that user might be confused if the grafana pod restarts and data lost, once user modified the grafana data such as the dashboards.

Reply:
I agree there is a chance for confusion. However, the official Grafana helm chart disables persistence by default, so I think we should too.

https://github.com/grafana/helm-charts/blob/2a422f6c43c3db33d31fc12d23957d5b6ff5e8c6/charts/grafana/values.yaml#L275-L277

Reply:
I agree, change default persistence value to false.